bridgekeeper/rules.py

```diff
@@ -320,7 +320,15 @@ def check(self, user, instance=None):
             # Find the appropriate LHS on this object, traversing
             # foreign keys if necessary.
             lhs = instance
-            for key_fragment in key.split("__"):
+            fragments = key.split("__")
+            for i, key_fragment in enumerate(fragments):
+                # Catch a many-to-many or many-to-one traversal and split it
+                # across multiple Rules
+                if not hasattr(lhs.__class__, "_meta"):
```

Comment:
I suspect there is a better way to detect this (like possibly even just `if isinstance(lhs, Manager):` as is used further down